corehq/apps/hqwebapp/static/hqwebapp/js/select2_handler_v4.js

```diff
@@ -16,6 +16,7 @@ hqDefine("hqwebapp/js/select2_handler_v4", [
         var self = {};
         self.fieldName = options.fieldName;
         self.placeholder = options.placeholder;
+        self.allowClear = options.allowClear;
```

Comment:
Is this change related to the SMS changes? I don't see where it's being used. Also should it be defaulted to `true` to keep the same behavior?

Reply:
Yes, On sms setting page option clearing is not allowed.
I also thought of making default to true. But looking that this(v4) is only used only 1 another place where it was not very reasonable to have  default `true` therefore I did not make it default. 
Rethought on this you are right i make sense to make it default else other new places using(v4) have to explicitly set it `true`.
Making it default